.github/workflows/functional-tests.yaml

```diff
@@ -0,0 +1,82 @@
+name: Functional Tests
+
+on:
+  workflow_run:
+    workflows: [Functional Tests Approval]
+    types:
+      - completed
+
+env:
+  # Go version to install
+  GOVER: '^1.22'
+  GOPROXY: https://proxy.golang.org
+  
+  # gotestsum version - see: https://github.com/gotestyourself/gotestsum
+  GOTESTSUMVERSION: 1.10.0
+
+  # github contexts
+  GH_RUN_ID: ${{ github.run_id }}
+  GH_RUN_NUMBER: ${{ github.run_number }}
+  GH_RUN_ATTEMPT: ${{ github.run_attempt }}
+
+  # Cloud account fields
+  INVISINETS_GCP_PROJECT_PARENT: ${{ secrets.GCP_FOLDER_ID }}
```

Comment:
Can the same lines be removed from build.yaml since it isn't using the clouds at all any more?

Reply:
Good point